pending_code_changes.txt

```diff
@@ -1 +1 @@
-
+ - Ported ShortNumberInfo to Javascript.
```

Comment:
Please include 2 lines as suggested.
Missing the first line:
`Code changes:`

(we group changes e.g. into `Code changes` and `Build changes`, for this change here we just highlight the former)

Reply:
Done!